### PR TITLE
[mce-dbus] Identify clients behind xdg-dbus-proxy

### PR DESCRIPTION
**Pull request by spiiroin**
Display blanks while recording video with lockscreen camera. This
happens because regular dbus client identification sees pid of
xdg-dbus-proxy process associated with the sandbox in whic the camera
application is running rather than pid of the camera application
itself and this does not match the topmost window pid reported by
lipstick.

Check client pid obtained via GetConnectionUnixProcessID. If it is
xdg-dbus-proxy, make an additional Identify call to query client
details from the proxy.

Signed-off-by: Simo Piiroinen <simo.piiroinen@jolla.com>

### CODE REVIEW

File: mce-dbus.c [hunk 1]
```diff
@@ -1276,6 +1302,29 @@ peerinfo_set_owner_name(peerinfo_t *self, const char *name)
...
+peerinfo_get_proxy_pid(const peerinfo_t *self)
+{
+    return self->pi_proxy_pid;
+}
+
+static void
+peerinfo_set_proxy_pid(peerinfo_t *self, pid_t pid)
+{
+    if( self->pi_proxy_pid == pid )
+	goto EXIT;
```
Comment (Tomin1) [inline mce-dbus.c:1315]: There is a weird mix of indentation with spaces and tabs in this change. For example this line is indented with one tab and line above has four spaces but I'm not sure what was the intention.

Comment (spiiroin) [reply mce-dbus.c:1315]: It is just what "tab=8 indent=4" looked like until people started messing with tab sizes. So, in context of this particular file it is intentional.

Comment (Tomin1) [reply mce-dbus.c:1315]: Okay :sweat_smile:

**Review by inzanity — APPROVED** [review]

**Comment by spiiroin** [comment]
D-Bus policy config needs to be elsewhere -> dropped from mce pull req.

**Pull request merged**